_faq/general/updating-documentation.md

```diff
@@ -0,0 +1,14 @@
+---
+title: How do I update the documentations?
+order: 2
+---
+{::options auto_ids="false" /}
+
+Forseti documentations can be updated in the `forsetisecurity.org-dev` branch.
+
+You can test your documentation changes locally, by
+[following the instructions here]({% link _docs/latest/develop/website.md %}).
+
+After your updated documentation is merged, the Forseti team will merge it
```

Comment:
Should we specify the timeline of that happening, maybe per release?

Reply:
Done